f600233d1b154d507b782f84604bcb12/update/sys_hub_action_type_definition_ae1e62bedb1a59108c045e8cd396190a.xml

```diff
@@ -1242,13 +1230,12 @@
         <sys_created_by>jeffsegal</sys_created_by>
         <sys_created_on>2023-05-05 15:26:04</sys_created_on>
         <sys_id>bc692d361be261103f288733cc4bcbb0</sys_id>
-        <sys_mod_count>11</sys_mod_count>
+        <sys_mod_count>18</sys_mod_count>
         <sys_updated_by>1370228783.CTR</sys_updated_by>
-        <sys_updated_on>2023-05-25 21:56:52</sys_updated_on>
+        <sys_updated_on>2023-06-22 15:03:37</sys_updated_on>
         <value>(function execute(inputs, outputs) {
   var getPalt = function() {
-    var ap = new GlideRecord("x_g_dis_atat_acquisition_package");
-    ap.get(inputs.acquisition_package);
+    var ap = inputs.acquisition_package;
```

Comment:
👍 

Original code written as though `inputs.acquisition_package` contains a 32 character sys_id, but is actually an object already passed in to the SNOW action